docs/learning/transformer/getting-started.md

```diff
@@ -0,0 +1,87 @@
+# Getting Started with iModel-Transformer
+
+## Overview
+
+The repository supports both JavaScript and TypeScript. To build a strong foundation, check out these resources:
+
+- **JavaScript:**  
+  [The Modern JavaScript Tutorial](https://javascript.info/)  
+  [Mozilla JavaScript Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference)
+  
+- **TypeScript:**  
+  [TypeScript Documentation](https://www.typescriptlang.org/docs/)
+
+For more details on the transformer itself:
+
+- **Transformer Repo:** [GitHub - imodel-transformer](https://github.com/iTwin/imodel-transformer)
+- **Transformer Docs:** [iTwinJS Transformer Learning](https://www.itwinjs.org/learning/transformer)
+- **Branching & Synchronization:** [Branching iModels Documentation](https://github.com/iTwin/imodel-transformer/blob/main/docs/learning/transformer/branching-imodels.md)
```

Comment:
This should be a relative link since the target exists within the trasnformer docs. You can use `$docs` to represent the root of the doc site (i.e. `$docs/learning/trasnformer/branching-imodels.md`).